app/controllers/admin_public_body_controller.rb

```diff
@@ -142,10 +142,12 @@ def mass_tag_add
   def missing_scheme
     # There might be a way to do this in ActiveRecord, but I can't find it
     @public_bodies = PublicBody.find_by_sql("
-            SELECT a.id, a.name, a.url_name, COUNT(*) AS howmany
```

Comment:
hard to say for sure without running the entire test suite but the controller spec doesn't catch it so, at it's simplest, something like...

```rb
  describe 'GET #missing_scheme' do

    it 'returns successfully' do
      get :missing_scheme
      expect(response).to be_success
    end

  end
```

added to `spec/controllers/admin_public_body_controller_spec.rb` will trip it up